.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- Please take a look at our Contributing documentation before submitting a Pull Request!
+https://rook.io/docs/rook/latest/Contributing/development-flow/
+
+Thank you for contributing to Rook! -->
+
+**Description of your changes:**
+
+**Which issue is resolved by this Pull Request:**
+Resolves #
+
+**Checklist:**
+
+- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
```

Comment:
If we're going to ask for the same structure, let's enable the commitlint bot and copy over the same .commitlintrc.json to get started.

Reply:
copied from rook and trimmed the commit title list.